theseus_gui/src/components/ui/UserSection.vue

```diff
@@ -0,0 +1,45 @@
+<script setup>
+import { Avatar } from 'omorphia'
+</script>
```

Comment:
Do all of these need to be there own components? mainly: `AppBar`, `News`, `UserSection`. they seem to be only used once in one part of the layout

Reply:
Reopening these- seems like this doesn't have to be it's own component